src/services/openletters/index.jsx

```
@@ -470,6 +471,67 @@ export const AddOpenLetter = ({
   );
 };
 
+
+export const DeleteOpenLetter = ({
+  setDeleteModal,
```

ESLint

[error] 'setDeleteModal' is missing in props validation

(react/prop-types)